packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -273,6 +279,9 @@ abstract class ResidentRunner {
       // F10, exit
       await stop();
       return true;
+    } else if (lower == 'd') {
```

Comment:
Apps on Fuchsia don't shut down cleanly after a 'q'. This is useful maybe only until I track that down and/or while bringing up hot reload for Fuchsia. I'll leave this here and remove the help message.